contributing/lts-release.md

```diff
@@ -0,0 +1,33 @@
+# Long-Term Stable (lts) Release Channel
+
+The **lts** release channel provides a previous **stable** build for four-week intervals. Approximately every four weeks, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel.
```

Comment:
the first sentence says "four-week intervals" second says "approximately every four weeks."  With the "second Monday" approach we'd actually be at either 4 weeks or 5 weeks (e.g. 4 weeks between Feb 10 and March 9, 5 weeks between March 9 and April 13).

Maybe just go with "approximately monthly"?

Reply:
Per above comment, since there's a handful of follow-up PRs/changes prompted by (and blocked by) this PR, my plan is to submit this then immediately follow-up with a PR updating these docs and others with the specifics of the LTS release schedule. For now, I'll replace this with approximately monthly, but know there will be a followup with tweaks.